### ISSUE TITLE
Could not get item in TypedDict: 'str' is not a string literal

### ISSUE THREAD

**Issue by bfelbo** [microsoft/pyright#424]
**Describe the bug**
When referencing a key in TypedDict using a string variable rather than a string literal, pyright raises the following issue: `Could not get item in TypedDict: 'str' is not a string literal`

**To Reproduce**
Run code below

**Expected behavior**
No error

**Screenshots or Code**
```
from typing_extensions import TypedDict

UserType = TypedDict("UserType", {"bio": str, "name": str},)
user: UserType = {"bio": "Sup.", "name": "Bob"}

for key in ["bio", "name"]:
    print(user[key])
```

**VS Code extension or command-line**
VS Code extension

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.1.11, which I just published.
